src/form.js

```diff
@@ -802,7 +802,32 @@ define([
             }
         },
         changeMugType: function (mug, questionType) {
-            this.mugTypes.changeType(mug, questionType);
+            var _this = this;
```

Comment:
nit: I don't think you need to use `_this`, because you aren't changing contexts anywhere in this function